test/acceptance/accessibility.py

```diff
@@ -243,6 +243,78 @@ def test_multiple_rubrics(self):
         self._check_a11y(self.staff_area_page)
 
 
+class MultipleOpenAssessmentA11yTest(OpenAssessmentA11yTest, FullWorkflowMixin):
+    """
+    Test accessibility when we have a unit containing multiple ORA blocks.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentA11yTest, self).setUp('multiple_ora', staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
```

Comment:
This is interesting. self.staff_area_page, and all the other page objects, will only be for 1 of the 2 problems. That's fine in this case because the accessibility check actually verifies the entire page, not just the HTML within the page object whose _check_a11y  method is being called.